mining.go

```diff
@@ -673,19 +669,19 @@ func maybeInsertStakeTx(bm *blockManager, stx *dcrutil.Tx, treeValid bool) bool
 // work off of is present, it will return a copy of that template to pass to the
 // miner.
 // Safe for concurrent access.
-func handleTooFewVoters(subsidyCache *standalone.SubsidyCache, nextHeight int64, miningAddress dcrutil.Address, bm *blockManager) (*BlockTemplate, error) {
-	timeSource := bm.cfg.TimeSource
-	stakeValidationHeight := bm.cfg.ChainParams.StakeValidationHeight
+func (g *BlkTmplGenerator) handleTooFewVoters(subsidyCache *standalone.SubsidyCache, nextHeight int64, miningAddress dcrutil.Address) (*BlockTemplate, error) {
```

Comment:
The `subsidyCache` can be fetched from `g`, `handleFewVoters` func sig can be simplified to `handleTooFewVoters(nextHeight int64, miningAddress dcrutil.Address) (*BlockTemplate, error)` as a result.